lib/cli.js

```diff
@@ -877,6 +877,7 @@ exports.search = function(keywords) {
 
 exports.init = function(options) {
   if (!shell.which('git')) return console.log(chalk.red('    Prerequisite not installed: git'));
+  if (!shell.which('mongod')) return console.log(chalk.red('    Prerequisite not installed: mongod'));
```

Comment:
shelljs is supposed to be cross platform and mongod.exe executes as mongod from cmd in Windows, but I didn't test to see that this worked properly. I'll let you know as soon as I verify that.